text/0000-libtest-json-output.md

```diff
@@ -0,0 +1,119 @@
+- Feature Name: libtest_json_output
+- Start Date: 2017-12-04
+- RFC PR:
+- Rust Issue:
+
+# Summary
+[summary]: #summary
+
+Add a machine-readable JSON-output mode for Rust's libtest.
+
+# Motivation
+[motivation]: #motivation
+
+Adding a machine-readable output for Rust's built-in tests will make external tool integration easier.
+
+Having this feature is not intended to replace the proposed custom test runners, but to enrich
+the default set of features offered by rust out-of-the-box.
+
+# Guide-level explanation
+[guide-level-explanation]: #guide-level-explanation
+
+Test binaries compiled `cargo test` or `rustc --test` allow the user to specify an output format using the `--format` flag.
+
+Using this flag, the format can be selected from one of the following:
+ - `pretty`: The default setting; print test results in a detailed manner.
+ - `terse`: Equivalent to `-q`; display one character per test instead of one line.
```

Comment:
Is this a new addition as well?

Reply:
Not really, it's just the old -q that libtest already has.
It was present in the PR as well.